src/main/java/org/maxgamer/quickshop/util/language/game/MojangGameLanguageImpl.java

```diff
@@ -69,19 +75,25 @@ public MojangGameLanguageImpl(@NotNull QuickShop plugin, @NotNull String languag
         if ("default".equalsIgnoreCase(languageCode)) {
             Locale locale = Locale.getDefault();
             languageCode = locale.getLanguage() + "_" + locale.getCountry();
+            if (StringUtils.isEmpty(locale.getCountry())) {
+                languageCode = locale.toLanguageTag();
+                plugin.getLogger().info("Your system offer empty data about locale country, we will fallback to system language default country, current language set to " + languageCode + ". If it incorrect, please edit it in config.yml.");
```

Comment:
What about `locale.getLanguage()` return ""?

Reply:
so it will broken, but i didn't found better way.